uniffi_bindgen/src/interface/conventions.rs

```diff
@@ -0,0 +1,127 @@
+use std::collections::{BTreeMap, BTreeSet};
+use uniffi_meta::Type;
+use crate::{CodeOracle, ComponentInterface, Renameable};
+use crate::interface::{FfiDefinition, Record};
+
+impl ComponentInterface {
+    pub fn apply_naming_conventions<O: CodeOracle>(&mut self, oracle: O) {
+
+        for (_, t) in &mut self.types.type_definitions {
+
+            if t.name().is_some() {
+                t.rename(oracle.var_name(&t.name().unwrap()));
```

Comment:
I think this should be either `class_name` since it's usually something like a record/enum/interface name.

At first I thought it should be a match statement that handled different variants differently, but AFAICT it's always a class name.